app/src/main/java/org/oppia/app/screenshots/ScreenshotManager.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.app.screenshots
```

Comment:
This should be part of the testing module--we don't want this included in the production build of the app.

Reply:
Moved to the `testing` module